### ISSUE TITLE
[Bug] Halo96v2 Num Lock Indicator not working

### ISSUE THREAD

**Issue by calirng** [ryodeushii/gem80-qmk-firmware#11]
### Describe the Bug

Pressing Num Lock is supposed to turn on the LED under the NUM key on Halo96v2, but doesn't due to incorrect column index.

Bug found in ansi.c

```
        if (showNumLock) {
            rgb_matrix_set_color(get_led_index(1, 14), 0x00, 0x80, 0x00);
        }
```
Fix:

```
        if (showNumLock) {
            rgb_matrix_set_color(get_led_index(1, 15), 0x00, 0xff, 0xff);
        }
```

### Keyboard Used

Nuphy Halo96v2

### Is AutoHotKey / Karabiner installed

- [ ] AutoHotKey (Windows)
- [ ] Karabiner (macOS)

**Comment by calirng** [comment]
Fixed and committed change

**Comment by ryodeushii** [comment]
fixed in #13

**Comment by ryodeushii** [comment]
@calirng and again, thanks for putting effort into improvements :)
